discretisedfield/region.py

```diff
@@ -640,16 +640,16 @@ def multiplier(self):
         """Compute multiplier for the region."""
         return uu.si_max_multiplier(self.edges)
 
-    def __mul__(self, other):
-        """Binary ``*`` operator.
+    def __mul__(self, factor):
+        """Scale the region.
 
         It can be applied only between ``discretisedfield.Region`` and
         ``numbers.Real``. The result is a region whose ``pmax`` and ``pmin``
         are multiplied by ``other``.
 
         Parameters
         ----------
-        other : numbers.Real
+        other : array-like or numbers.Real
```

Comment:
Currently we are allowing `pmin` and `pmax` to be `numbers.Number` so I think we could allow factor to be the same type